azure/durable_functions/grpc/protobuf/DurableRpc.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+service DurableRpc {
+    rpc ClientOperation (ClientRequest) returns (ClientResponse) {}
+}
+
+message ClientRequest {
+    enum Type {
+        Status = 0;
+        Create = 1;
+        SendEvent = 2;
+        Terminate = 3;
+        Rewind = 4;
+        Purge = 5;
+    }
+
+    // Type of client request.
+    Type type = 1;
+
+    // properties
+    map<string, RequestData> args = 2;
+}
+
+message ClientResponse {
+    enum Result {
+        Completed = 0;
+        Failed = 1;
+    }
+
+    // 
+    Result result = 1;
+
+    //
+    bytes error = 2;
+
+    // TODO: not bytes
+    ResponseData data = 3;
+}
+
+// Used to encapsulate request data which could be a variety of types
+message RequestData {
+    oneof data {
+        string string = 1;
```

Comment:
@kashimiz  shall we change the naming convention for properties to include type but also another keyword such as value - `stringValue` for example? Thank you!

Reply:
This is minor so I think we can merge it that way and if want to improve it we can do it later. Good job!